include/nodes/internal/DataModelRegistry.hpp

```diff
@@ -57,11 +60,16 @@ class NODE_EDITOR_PUBLIC DataModelRegistry
 
     QString const name = uniqueModel->name();
 
-    if (_registeredModels.count(name) == 0)
+    if (!_registeredModels.count(name))
```

Comment:
You might find this less readable, but I've seen people treating the `count` method on `std::map` or `std::set` as "contains", which is why I wrote is as `!_registeredModels.count(name)` here.